plugins/modules/rds_cluster_param_group.py

```diff
@@ -0,0 +1,369 @@
+        tags:
+            description: dictionary of tags
+            type: dict
+            returned: when state is present
+"""
+
+from itertools import zip_longest
+
+try:
+    import botocore
+except ImportError:
+    pass  # Handled by AnsibleAWSModule
+
+from ansible.module_utils.common.dict_transformations import camel_dict_to_snake_dict
+from ansible.module_utils.common.dict_transformations import snake_dict_to_camel_dict
+
+from ansible_collections.amazon.aws.plugins.module_utils.botocore import is_boto3_error_code
+from ansible_collections.amazon.aws.plugins.module_utils.modules import AnsibleAWSModule
+from ansible_collections.amazon.aws.plugins.module_utils.retries import AWSRetry
+from ansible_collections.amazon.aws.plugins.module_utils.tagging import ansible_dict_to_boto3_tag_list
+from ansible_collections.amazon.aws.plugins.module_utils.tagging import boto3_tag_list_to_ansible_dict
+from ansible_collections.amazon.aws.plugins.module_utils.tagging import compare_aws_tags
+
+
+@AWSRetry.jittered_backoff()
+def _describe_db_cluster_parameters(module, connection, group_name):
+    try:
+        paginator = connection.get_paginator("describe_db_cluster_parameters")
+        return paginator.paginate(DBClusterParameterGroupName=group_name).build_full_result()["Parameters"]
+    except is_boto3_error_code("DBParameterGroupNotFound"):
+        return None
+    except botocore.exceptions.ClientError as e:  # pylint: disable=duplicate-except
+        module.fail_json_aws(e, msg="Failed to describe existing RDS cluster parameter groups")
+
+
+def modify_parameters(module, connection, group_name, parameters):
+    if module.check_mode:
+        return True
+
+    current_params = _describe_db_cluster_parameters(module, connection, group_name)
+    parameters_names = [x["parameter_name"] for x in parameters]
+    parameters = snake_dict_to_camel_dict(
+        list(map(lambda x: {k: v for k, v in x.items() if v is not None}, parameters)), capitalize_first=True
+    )
+    for chunk in zip_longest(*[iter(parameters)] * 20, fillvalue=None):
+        non_empty_chunk = [item for item in chunk if item]
+        try:
+            connection.modify_db_cluster_parameter_group(
+                aws_retry=True, DBClusterParameterGroupName=group_name, Parameters=non_empty_chunk
+            )
+        except (botocore.exceptions.ClientError, botocore.exceptions.BotoCoreError) as e:
+            module.fail_json_aws(e, msg="Couldn't update RDS cluster parameters")
+    update_params = _describe_db_cluster_parameters(module, connection, group_name)
+    changed = False
+    for name in parameters_names:
+        previous = list(filter(lambda x: x["ParameterName"] == name, current_params))[0]
+        new = list(filter(lambda x: x["ParameterName"] == name, update_params))[0]
+        if new != previous:
+            changed = True
+            break
+    return changed
+
+
+def _list_resource_tags(module, connection, resource_arn):
```

Comment:
You should use  the get_tags utility function https://github.com/ansible-collections/amazon.aws/blob/main/plugins/module_utils/rds.py#L357